invenio_rdm_records/resources/serializers/dublincore/__init__.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Dublin Core Serializers for Invenio RDM Records."""
+
+from dcxml import simpledc
+from flask_resources.serializers import MarshmallowJSONSerializer, \
+    SerializerMixin
+
+from .schema import DublinCore
+
+
+class DublinCoreJSONSerializer(MarshmallowJSONSerializer):
+    """Marshmallow based Dublin Core serializer for records."""
+
+    def __init__(self, **options):
+        """Constructor."""
+        super().__init__(schema_cls=DublinCore, **options)
+
+    def dump_one(self, obj):
```

Comment:
I think it does not defer on implementation from the [parent class](https://github.com/inveniosoftware/flask-resources/blob/master/flask_resources/serializers/json.py#L92), I think is not needed :)